src/common/playlists/PlayList.jsx

```diff
@@ -23,7 +26,7 @@ const PlayList = () => {
     return (
       <div className='play-not-found'>
         <ImageOops className='play-not-found-image' />
-        <p className='page-404-lead'>Play not found</p>
+        <p className='page-404-lead'>Play not found for "{ searchTerm }" </p>
```

Comment:
@vasantisuthar , Can we add check here that show `for " searchTerm" ` only if searchTerm is not a falsy value?

Reply:
Hey @vasantisuthar, Can you address the review comment?

Reply:
> @vasantisuthar , Can we add check here that show `for "searchTerm" ` only if searchTerm is not a falsy value?

I didn't get this, can you please elaborate?

Reply:
Sure, @vasantisuthar , we need to handle the case when `searchTerm` is a false value like empty, undefined, etc.
So if `searchTerm` is a falsy value then we should just show `Play not found`.

Reply:
okay

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Readable